src/api/database/misc.rst

```diff
@@ -17,40 +17,22 @@
 ==============
 
 .. http:post:: /{db}/_purge
-    :synopsis: Purges some historical documents entirely from database history
+    :synopsis: Purges documents entirely from database
 
-    A database purge permanently removes the references to deleted documents
-    from the database. Normal deletion of a document within CouchDB does not
+    A database purge permanently removes the references to documents
+    in the database. Normal deletion of a document within CouchDB does not
     remove the document from the database, instead, the document is marked as
     ``_deleted=true`` (and a new revision is created). This is to ensure that
     deleted documents can be replicated to other databases as having been
     deleted. This also means that you can check the status of a document and
     identify that the document has been deleted by its absence.
 
-    .. warning::
-        Purging a document from a database should only be done as a last resort
-        when sensitive information has been introduced inadvertently into a
-        database. In clustered or replicated environments it is very difficult
-        to guarantee that a particular purged document has been removed from
-        all replicas. Do not rely on this API as a way of doing secure
-        deletion.
+    The format of the purge request must include the document IDs and for each
```

Comment:
I suggest removing "format" from this sentence. Instead, just: "The purge request must..."